stable/database/files/nuosm

```diff
@@ -22,12 +22,13 @@ NUODB_BINDIR=$NUODB_HOME/bin
 [ -z "$NUOCMD" ] && NUOCMD="$NUODB_BINDIR/nuocmd --api-server $NUOCMD_API_SERVER"
 
 # attempt to retain the previous crash directory (within the configured window to avoid filling the disk)
-if [ -f $NUODB_CRASHDIR/* ]; then
-  crashcount=$(find $NUODB_LOGDIR/crash-* -type d -cmin -$OVERWRITE_WINDOW ! -path . | wc -l)
-  if [ $crashcount -lt $OVERWRITE_COPIES ]; then
+crashcount=$(find $NUODB_CRASHDIR/core* -maxdepth 0 ! -type d 2>/dev/null | wc -l)
+if [ $crashcount -ge 1 ]; then
+  retainedcrashcount=$(find $NUODB_LOGDIR/crash-* -maxdepth 0 -type d -cmin -$OVERWRITE_WINDOW 2>/dev/null | wc -l)
+  if [ $retainedcrashcount -lt $OVERWRITE_COPIES ]; then
     crashbackupdir="$NUODB_LOGDIR/crash-$( date +%Y%m%dT%H%M%S )/"
     mkdir $crashbackupdir
-    mv $NUODB_CRASHDIR/* $crashbackupdir
+    mv $NUODB_CRASHDIR/core* $crashbackupdir
```

Comment:
This will work with manually created cores via `nuocmd get core-file` (file name: `core.nuodb.demo.sm-database-nuodb-amazon0-t24-hotcopy-0.0.gz`). Thanks.

Reply:
OK - although it would only process them on SM restart of course.